### PR TITLE
Feature : Adding cards in hand number

### PR DESCRIPTION
**Pull request by pierrehilbert**
Not knowing the number of cards in the opponent hand is a problem to evaluate his possibilities.
With this feature, you will be able to see the number of cards without exposing which cards he has..

### CODE REVIEW

File: src/Player.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ export class Player {
     public plants: number = 0;
     public plantProduction: number = 0;
     public cardsInHand: Array<IProjectCard> = [];
+    public cardsInHandNbr: number = 0;
```
Comment (bafolts) [inline src/Player.ts]: The `cardsInHandNbr` property isn't needed on `Player`. The rest looks great.

File: src/components/OtherPlayer.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export const OtherPlayer = Vue.component("other-player", {
             <h3 :style="'color:' + player.color" v-on:click="toggleDisplayed()">{{player.name}}</h3>
             <div v-if="displayed === true">
                 <div>Terraform Rating: {{player.terraformRating}}</div>
+                <div>Cards In Hand: {{player.cardsInHandNbr}}</div>
```
Comment (alrusdi) [inline src/components/OtherPlayer.ts]: We can say just:
{{ player.cardsInHand.length }}

Comment (pierrehilbert) [reply src/components/OtherPlayer.ts]: No, the attribute wasn't available for other players (take a look at the server.ts modification).
Having this new attribute allow to have the number of cards without being able to see which one in the console.

Comment (alrusdi) [reply src/components/OtherPlayer.ts]: Oh, you right! My fault.

**Pull request merged**